development-defaults.ini

```diff
@@ -41,8 +41,6 @@ supporter_deadline = "2015-12-26"
 
 printed_badge_deadline = "2016-01-04"
 
-room_deadline = "2015-11-30"
```

Comment:
``room_deadline`` is still required to be in this file if you're using the ``hotel`` plugin.  So we could remove it here, but only if we're sure that puppet will still drop it in place on our staging and production servers and also vagrant.